frontend/src/components/input-tag/index.tsx

```
@@ -50,86 +29,24 @@ export const TagLine: React.SFC<TagLineProps> = ({
     },
   );
 
-  if (loading) {
+  if (loading || error) {
```

we should probably throw the error :)

---

ah yes, I wanted to change it but temporarily used `|| error`

---

done!